test/unit/object/test_base_api_json_object.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+import pytest
+
+from boxsdk.object.base_api_json_object import BaseAPIJSONObject
+
+
+@pytest.fixture(params=[{'foo': 'bar'}, {'a': {'b': 'c'}}])
+def base_api_json_object(request):
+    return request.param, BaseAPIJSONObject(request.param)
```

Comment:
These could be two separate fixtures. One for the vanilla `dict`, and then another (depending on the first) which is the `BaseAPIJSONObject`. And then use both of those fixtures in the test method.